src/core/algorithm/plan.cc

```diff
@@ -92,6 +95,16 @@ void iterateAvailablePlansInternal(const CoreField& field,
         n = 10;
     }
 
+    if (eventIndex < events.size()) {
+        const Plan::Event& event = events[eventIndex];
+        if (event.type == Plan::Event::Type::FALL_OJAMA_ROWS && event.frames < totalFrames) {
+            totalFrames += field.fallOjama(event.value);
+            ++eventIndex;
```

Comment:
Could you comment TODO or something when we have two ojama-fall events in a row in totalFrames?
Maybe it should be merged before passing here, though.